docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,101 @@
+.. _ref_quickstart_modeling:
+
+=================
+Modeling the data
+=================
+
+.. edb:split-section::
+
+  The flashcards application has a simple data model, but it's interesting enough to get a taste of many of the features of the |Gel| schema language. You have a ``Card`` type that describes a single flashcard, which for now contains two required string properties: ``front`` and ``back``. Each ``Card`` belongs to a ``Deck``, and there is an explicit ordering to the cards in a given deck.
+
+  Looking at the mock data, you can see this structure in the JSON.
```

Comment:
maybe to mention here that we mean the file `deck-edgeql.json`?

Reply:
`decks.json` is where the data lives once you import the `deck-edgeql.json`, so I'd like to keep that rather than make the actions module more complicated by needing to check if it exists, etc.

> maybe to mention here that we mean the file deck-edgeql.json?

Yeah, that's a good call. Maybe show a bit of the data itself.